react/GlovoAdmin.tsx

```diff
@@ -0,0 +1,814 @@
+import type { FC } from 'react'
+import React, { useEffect, useState } from 'react'
+import { useMutation, useQuery } from 'react-apollo'
+import type { InjectedIntlProps } from 'react-intl'
+import { injectIntl } from 'react-intl'
+import { formatIOMessage } from 'vtex.native-types'
+import {
+  ButtonWithIcon,
+  Checkbox,
+  Collapsible,
+  InputPassword,
+  Input,
+  Layout,
+  PageBlock,
+  PageHeader,
+  Table,
+  Alert,
+  ModalDialog,
+  Button,
+  Card,
+} from 'vtex.styleguide'
+import styles from './GlovoAdmin.css'
+
+import Modal from './components/ModalGlovo'
+import APP_SETTINGS from './graphql/appSettings.graphql'
+import SAVE_APP_SETTINGS from './graphql/saveAppSettings.graphql'
+import iconGlovo from './icons/icon.png'
+import IconAdd from './icons/IconAdd'
+import IconDelete from './icons/IconDelete'
+import IconEdit from './icons/IconEdit'
+import { messageUI, NameFields } from './shared'
+import type { SettingDataType, AffiliationType, TableItem } from './shared'
+
+const GlovoAdmin: FC<InjectedIntlProps> = ({ intl }) => {
+  const edit = <IconEdit />
+  const deleteItem = <IconDelete />
+  const add = <IconAdd />
+
+  const [isOpen, setIsOpen] = useState(false)
+  const [cards, setCards] = useState([])
+  const [alertConfig, setAlertConfig] = useState(false)
+  const [alertAff, setAlertAff] = useState(false)
+  const [itemDelete, setItemDelete] = useState('')
+  const [alertDialog, setAlertDialog] = useState(false)
+  const [isOpenModal, setIsOpenModal] = useState(false)
+  const [typeModal, setTypeModal] = useState(NameFields.TYPENEW)
+  const [fromMobile, setFromMobile] = useState(false)
+  const [dataEdit, setDataEdit] = useState<AffiliationType>({
+    id: '',
+    nameAffiliation: '',
+    storeId: '',
+    salesChannel: '',
+    pickupPoints: '',
+    glovoId: '',
+  })
+
+  const [settingsState, setSettingsState] = useState<SettingDataType>({
```

Comment:
This will have to be updated once the data structure is modeled to match the one used i the backend.
I would suggest trying to use the same naming conventions to avoid confusion.
```
AffiliationType --> StoreInfo
```